SU-SWS/stanford_publication
# D8CORE-3722: Publication Module styling updates.

# READY FOR REVIEW

# Summary
- Edits to the display for the Publications module. 
- Make gutters match between filtered and not filtered page.
- Styling of the filtered results page (Publications | Results for: tag
- Added the filtered menu open and close. It starts closed on the responsive view. 
- Reduced space between the pub-type and the title.

# Needed By (Date)
- 3/23 - It would be nice to have earlier for more testing.

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. Make sure you've updated the profile as well to get a newly merged pull request from there.
3. Verify the menu open and closes. 
4. Verify it starts closed on Responsive but also opens and closes.
5. Verify the spacing b/t “publication type” and the title to MS(0), 18px 
6. Verify the gutter between the filter menu and content well is MS 6 and matches News.

# Affected Projects or Products
- Publications
- SBT SOE SAA

# Associated Issues and/or People
- D8CORE-3947
- D8CORE-3722
- D8CORE-3746
- D8CORE-3949
- @katrialesser 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
